wasmedge-ggml-llama-interactive/src/main.rs

```diff
@@ -44,6 +44,11 @@ fn main() {
         .trim()
         .parse()
         .unwrap();
+    let is_interactive: bool = env::var("is_interactive")
```

Comment:
This is not an option for the metadata, considering it as a parameter in the rust program.

Reply:
Okay, I will change the behavior to:

- If only the preload model name is given, then it defaults to interactive mode.
- If another argument is provided after the preload model name, then it will be considered as a prompt, and it will enter non-interactive mode (just like the previous wasmedge-ggml-llama example).

Reply:
Done.